46-Screen listado de novedades

ByMofm opened the pull request:
Se creo el componente Backoffice/News
-Se lista todo en una tabla
-Se crearon los botones para editar y eliminar

Commit:
First Commit

-Created component News
-update dependencies.

Commit:
Update component

Updated route components.

Commit:
Update App.js

Commit:
Updated

Updated App.js and Index.js

Commit:
Merge branch 'main' into 46-Screen-listado-de-novedades

Commit:
Update branch from main

Actualice la rama para poder solucionar todos los conflictos del package-lock .
-Tambien movi el componente Footer para la misma carpeta que el resto de los componentes, no se si correspondia por aca , cualquier cosa lo realizo en un issue nuevamente.

Galaeno commented:
@ByMofm tenés conflictos. Agregá descripción al PR

ByMofm committed:
Merge branch 'main' into 46-Screen-listado-de-novedades

Galaeno commented on src/components/backoffice/news.js:
```diff
@@ -0,0 +1,110 @@
+    };
+
+
+const newsData = useMemo(() => [...values], [values]);
```
Está mal la indentacion, revisá todo el archivo y mejoralo por favor

ByMofm replied:
Corregido, disculpa la verdad si me habia quedado bastante desprolija la indentacion.

Commit:
Update component

-Se corrigió y mejoro la indentacion.

Commit:
Merge branch '46-Screen-listado-de-novedades' of https://github.com/alkemyTech/OT173-client-js into 46-Screen-listado-de-novedades

Galaeno commented on src/components/backoffice/news.js:
```diff
@@ -0,0 +1,92 @@
+
+  const loadValues = async () =>{
+    const response = await axios.get('/backoffice/news').catch(err => console.log(err));       
+      if(response){
```
Quedó mal la indentacion

ByMofm replied:
Corregido.

Galaeno commented on src/components/backoffice/news.js:
```diff
@@ -0,0 +1,92 @@
+  const newsData = useMemo(() => [...values], [values]);    
+
+  const loadValues = async () =>{
+    const response = await axios.get('/backoffice/news').catch(err => console.log(err));       
```
Si usas await, no deberias usar catch, pero tampoco deberias usar axios aca, sino deberias llamar al service

ByMofm replied:
Corregido, se añadio la utilizacion del servicio.

Galaeno commented on src/components/backoffice/news.js:
```diff
@@ -0,0 +1,92 @@
+    loadValues()
+  },[]);
+
+return(
```
Está mal la indentacion

ByMofm replied:
Corregido.

Commit:
Update

-Actualizado ruta BackofficeNews dentro del app.js
-Corrección de requerimiento para no listar en la tabla el "id"
-Corrección indentacion

Galaeno commented:
@ByMofm tenés conflictos

ByMofm committed:
Merge branch 'main' into 46-Screen-listado-de-novedades

Galaeno commented:
@ByMofm tenés conflictos

ByMofm committed:
Merge branch 'main' into 46-Screen-listado-de-novedades

ByMofm commented:
> @ByMofm tenés conflictos

Solucionados.
